skeleton/techdoc/docs/rhoai.md

```diff
@@ -12,16 +12,27 @@ This document will outline how you can run build and run your sample application
 
 ## Running the Sample
 
-1) Navigate to the OpenShift AI workbench created for your sample application
+1) While on Console, navigate to the `OpenShift AI`
+
+![image](./.assets/access-openshift-ai.png)
+
+2) Go to the `Data Science Projects` section and access your application's project
+
+![image](./.assets/data-science-projects.png)
+
+3) Access the `workbench`
```

Comment:
The templated workbench name is `{{ values.name }}-notebook`, it might be useful to include that here?